newIDE/app/src/UI/User/UserChip.js

```diff
@@ -34,7 +34,11 @@ const UserChip = ({ onOpenProfile }: Props) => {
     onOpenCreateAccountDialog,
     loginState,
   } = authenticatedUser;
-  const displayNotificationBadge = hasPendingNotifications(authenticatedUser);
+  // TODO: Remove the badge on the user chip and handle badge notifications
```

Comment:
still a todo?

Reply:
For later, I'm not sure how these 2 will interact